x-pack/elastic-agent/pkg/agent/cmd/container.go

```diff
@@ -80,6 +80,7 @@ The following actions are possible and grouped based on the actions.
 
   KIBANA_FLEET_HOST - kibana host to enable create enrollment token on [$KIBANA_HOST]
   FLEET_TOKEN_NAME - token name to use for fetching token from Kibana. This requires Kibana configs to be set.
+  DEFAULT_FLEET_TOKEN_NAME - token name to use when [$FLEET_TOKEN_NAME] is not set. Defaults to "Default policy".
```

Comment:
Is this the enrollment token name? Would it be more intuitive if we allowed the `elastic/fleet-server` service account to read the agent policy API to match on Agent Policy names instead of token names?

Reply:
Oh, this isn't the right value that was added, now that I look at it. This should be `DEFAULT_FLEET_TOKEN_POLICY_NAME` instead.

We could definitely optimize this to do the policy discrimination on the kibana side. I'll open a follow up PR to handle that refactor.